src/main/kotlin/step3/Race.kt

```diff
@@ -0,0 +1,34 @@
+package step3
+
+import java.util.Random
+
+object Race {
+    private const val MIN_VALUE_TO_MOVE = 4
+
+    fun initiate(carNumber: Int): ArrayList<Car> {
+        val carList = ArrayList<Car>()
+        for (i in 1..carNumber) {
+            carList.add(Car(0))
+        }
+        return carList
+    }
+
+    fun makeTurn(cars: ArrayList<Car>): ArrayList<Car> {
+        for (car in cars) {
+            car.distance = getDistanceForEachCar(car.distance)
+        }
+        return cars
+    }
+
+    fun getDistanceForEachCar(distance: Int): Int {
```

Comment:
`Race` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun getDistanceForEachCar(distance: Int): Int {
```

Reply:
해당 함수는 RaceTest class에서 테스트를 진행하고 있습니다. private 으로 변경하면 테스트 코드 클래스에서 접근이 불가능할 것 같아요.

Reply:
테스트를 위해 메서드의 접근 제한자를 바꿔야 할 때는 아래의 글이 도움 될 거예요.
https://www.slipp.net/questions/253